### PR TITLE
Feature/navigatum localization

### PR DESCRIPTION
**Pull request by ctsk**
## Issue

This fixes the following issue(s):
- Resolves: #1510

## Screenshot

image
image

_Note how the names of the properties are properly translated_

## Why this is useful for all students

Makes the apps localization more consistent :)

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt [hunk 1]
```diff
@@ -110,7 +110,7 @@ class NavigationDetailsFragment : BaseFragment<Unit>(
     private fun showNavigationDetailsProperties(navigationDetails: NavigationDetails) {
         navigationDetails.properties.forEach { property ->
             val propertyRow = NavigationPropertyRowBinding.inflate(layoutInflater, binding.propsList, true)
-            propertyRow.propertyName.text = getTranslationForPropertyTitle(property.title)
+            propertyRow.propertyName.text = property.title
             propertyRow.propertyValue.text = property.value
         }
     }
```
Comment (ctsk) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt:116]: Perhaps someone with more android-fu can help me here. Is it possible to do this more elegantly with bindings, now that we don't do the translation anymore?

Comment (CommanderStorm) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt:116]: I dont know of a better way

**Review by CommanderStorm — APPROVED** [review]
Thanks for this contribution ❤️

File: app/src/main/java/de/tum/in/tumcampusapp/component/other/locations/LocationManager.kt [hunk 2]
```diff
@@ -274,7 +274,8 @@ class LocationManager @Inject constructor(c: Context) {
             var geo: Geo? = null
             if (searchResult != null && searchResult.sections.isNotEmpty() && searchResult.sections[0].entries.isNotEmpty()) {
                 val location = searchResult.sections[0].entries[0].id
-                val locationDetails = NavigaTumAPIClient.getInstance(mContext).getNavigationDetails(location)
+                // The language does not matter, as we only use the geo information from this request
+                val locationDetails = NavigaTumAPIClient.getInstance(mContext).getNavigationDetails(location, "de");
```
Comment (CommanderStorm) [inline app/src/main/java/de/tum/in/tumcampusapp/component/other/locations/LocationManager.kt:278]:
```suggestion
                val locationDetails = NavigaTumAPIClient.getInstance(mContext).getNavigationDetails(location, "de")
```

**Pull request merged**